src/System.Diagnostics.Process/src/System/Diagnostics/Process.Windows.cs

```diff
@@ -95,6 +95,106 @@ public void Kill()
             }
         }
 
+        /// <summary>
+        /// Makes a best-effort attempt to kill all descendant (child, grandchild, etc.) processes.
+        /// </summary>
+        private void KillTree()
```

Comment:
I'd make it so that this function takes PID and then calls itself recursively for each child

Reply:
Not sure exactly what you're looking for here...are you suggesting to inline the logic into `KillChildren(…)` into this method? 

Note: `KillChildren(…)` is called by both the Windows and Unix platform implementations of `KillTree()`. If it's in-lined here, it will also need to be in-lined into the Unix code (code duplication).

Reply:
sounds good in this case